tests/test_migrate_command.py

```diff
@@ -102,3 +108,48 @@ def test_migrate_command_for_real(django_db_setup_no_init, settings):
     # check if max applied version is target version
     assert (migrations.get_applied_versions(connections['no_init'])[-1] ==
             settings.NORTH_TARGET_VERSION)
+
+
+@pytest.mark.django_db
+def test_migrate_command_with_django_table(django_db_setup_no_init, settings):
+    """
+    This test simulate the case when a project was created to be used with django migrations table
+    Either because it didn't use septentrion-based django-north,
+    or a pre septentrion-base django-north version.
+    """
+    connection = connections['no_init']
+
+    # We begin with an empty database
+    assert migrations.get_current_version(connection) is None
+
+    # We simulate the setup of the database in the past, with a django_migrations table.
+    septentrion.migrate(
+        **{
+            "MIGRATIONS_ROOT": settings.NORTH_MIGRATIONS_ROOT,
+            "target_version": "1.0",
+            "SCHEMA_TEMPLATE": getattr(
+                settings,
+                "NORTH_SCHEMA_TPL",
+                migrations.schema_default_tpl),
+            "DBNAME": connection.settings_dict["NAME"],
+            "HOST": connection.settings_dict["HOST"],
+            "USERNAME": connection.settings_dict["USER"],
+            "PASSWORD": connection.settings_dict["PASSWORD"],
+            "TABLE": "django_migrations",
+            "VERSION_COLUMN": 'app',
+            "NAME_COLUMN": 'name',
+            "APPLIED_COLUMN": 'applied',
+            "CREATE_TABLE": False,
```

Comment:
(Nitpick: This whole blocks makes the test a bit harder to read. it could be in a fixture)